smtp-mail.cabal

```diff
@@ -2,7 +2,7 @@
 -- documentation, see http://haskell.org/cabal/users-guide/
 
 name:                smtp-mail
-version:             0.1.4.6
+version:             0.1.4.7
```

Comment:
Since this is a breaking change, we'll need to have a major version bump. This will be `0.2.0.0`.